service/frontend/dcRedirectionHandler.go

```diff
@@ -142,16 +152,22 @@ func (handler *DCRedirectionHandlerImpl) DescribeTaskList(
 	request *shared.DescribeTaskListRequest,
 ) (*shared.DescribeTaskListResponse, error) {
 
-	targetDC, err := handler.redirectionPolicy.GetTargetDataCenterByName(request.GetDomain())
-	if err != nil {
-		return nil, err
-	}
+	var apiName = "DescribeTaskList"
```

Comment:
Is there a way to get the name of function we are in, so that we dont have to spell this out?